robocop/checkers/naming.py

```diff
@@ -679,7 +703,13 @@ def visit_KeywordCall(self, node):  # noqa
             if len(node.data_tokens) < 2:
                 return
             token = node.data_tokens[1]
-            if token.type == Token.ARGUMENT and not token.value.isupper():
+            if not token.value:
```

Comment:
It says the code is not covered by tests. It's possible if you add something like below code to tests:

```
Set Global Variable
...
...    previous arg is empty now
```